backend/open_webui/routers/orq.py

```
@@ -0,0 +1,169 @@
+
+@router.get("/config")
+async def get_config(request: Request, user=Depends(get_admin_user)):
+    """Get ORQ configuration"""
+    return {
+        "ENABLE_ORQ_API": request.app.state.config.ENABLE_ORQ_API,
+        "ORQ_API_KEY": request.app.state.config.ORQ_API_KEY,
+    }
+
+@router.post("/config/update")
+async def update_config(
+    request: Request, form_data: ORQConfigForm, user=Depends(get_admin_user)
```

Ruff (0.12.2)

Unused function argument: `user`

(ARG001)